examples/reference/panes/HoloViews.ipynb

```diff
@@ -197,36 +540,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import holoviews.plotting.mpl\n",
-    "import holoviews.plotting.plotly\n",
+    "def get_plot(n):\n",
+    "    values = list(range(n))\n",
+    "    return hv.Scatter({\n",
+    "        \"x\": values, \"y\": values\n",
+    "    }, kdims=\"x\", vdims=\"y\", extents=(0,0,100,100)).opts(height=300, responsive=True)\n",
     "\n",
-    "hv_pane = pn.pane.HoloViews(dmap, backend='matplotlib')\n",
-    "hv_pane"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Please note that in a *server context* you will also have to set the matplotlib backend like below\n",
+    "widget = pn.widgets.IntSlider(value=50, start=1, end=100, name=\"Number of points\")\n",
+    "hv_pane = pn.pane.HoloViews(hv.DynamicMap(pn.bind(get_plot, widget)),)\n",
     "\n",
-    "```python\n",
-    "import matplotlib\n",
-    "matplotlib.use('agg')\n",
-    "```\n",
-    "\n",
-    "The ``backend``, like all other parameters, can be modified after the fact.  To demonstrate, we can set up a select widget to toggle between backends for the above plot:"
+    "pn.Column(widget, hv_pane)"
```

Comment:
I don't understand why the widgets of the `DynamicMap` are not shown here. Its quite inconsistant that the widgets are shown if the `DynamicMap` is created via HoloViews redim as shown in the top example. But not when created with Panel bind.

![image](https://github.com/holoviz/panel/assets/42288570/d2261a8d-1a69-4301-8b69-bbce054e9ee4)

What is the *model* for understanding this?

Wrapping a bound function in a DynamicMap has been a *secret* among a few super users for a very long time. Please help me explain why this is beneficial. Thanks.

Reply:
Good question, the reason is historical. Originally HoloViews always constructed widgets for the dimensions internally (even before Panel). When we added `streams` in HoloViews it became possible to bind other arguments to the functions on a `DynamicMap` but those were generally obtained from actual `Stream` objects, such as the `RangeXY` or `Tap`. At some point it became possible to directly bind parameters using the `Param` stream and as syntactic sugar it became possible for HoloViews to accept functions annotated with `depends` or `bind`, however we didn't go the step further and inspect what the bound parameters or dependencies were and then create widgets for them. I agree that you could argue that this should be done but I'll leave that as an exercise for later and put some explanation of the current behavior into the reference docs now.